### ISSUE TITLE
Plans for a desktop app?

### ISSUE THREAD

**Issue by Freeplayg** [uxbox/uxbox#1811]
**Is your feature request related to a problem? Please describe.**
Some would rather have a desktop app than being forced to use their browser with an internet connection.

**Describe the solution you'd like**
A cross-platform desktop app (For Linux, use Flatpak. As it's consistent across any distro).

**Comment by EvaMarco** [comment]
Hello @Freeplayg:

**Do you have plans to create a desktop app?**
Yes, we have plans to create a desktop app, but not in the short term. Part of our product value proposition is being browser-based and in order to create a desktop app, we will have to change a big part of our core code, that makes it compete with other current priorities.

**I need to work offline, Can I do it with Penpot?**
To achieve a complete offline app we will need to re-implement many backend code in frontend, such as running a headless browser, image and/or font conversion, assets storage, and much more. There are some technical implications that we must study well before we take that step. 
Still, if you really need to use Penpot offline you can always install it locally. Here is a [link](https://help.penpot.app/technical-guide/getting-started/) to a simple how-to.

It will be very helpful for us to know what your specific needs for a desktop app are, thanks in advance.

**Comment by jdittrich** [comment]
Would desktop app need to be a single executable or could it also be a locally run, pre-packaged server? Software like [duplicati](https://www.duplicati.com/) and [taguette](https://www.taguette.org/) do that and it is fine.

**Comment by niwinz** [comment]
This is already discussed here: https://github.com/penpot/penpot/discussions/475 where you have my response here https://github.com/penpot/penpot/discussions/475#discussioncomment-2340247

TLDR: it depends on what problem we want to solve, desktop app will not bring any value as is without developing a specific feature set that bring the real value (as example: offline working, single binary, etc..) but not all this features are easy to be implement or really relevant. I mean, "desktop app" concept means too many different things to different people.
